docs/guides/monitor/statsd.md

```diff
@@ -265,7 +265,7 @@ The `info` field supports `html`, embedding useful links and instructions in the
 
 ## Vendoring a new collector
 
-After all this hussle, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.
+After all this hustle, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.
```

Comment:
This was actually harder than it looks, because I apparently mostly know this word phonetically and I initially picked the wrong correction for it...

Reply:
This probably should be "hassle." That makes the most sense to me in context. @OdysLam?

Reply:
@kickoke halp, what do you think is better?

Reply:
> @kickoke halp, what do you think is better?

I think we should move away from all these words. "Hustle" seems like bragging, "hassle" seems like complaining. We should just go neutral on this and say "effort/work/decisions" or even just say "After all of this,"

Reply:
👍 +1 what @DShreve2 said :)

Reply:
@kickoke or @DShreve2 can you reword it and make a suggestion so we can commit it using GitHub UI?

Reply:
```suggestion
After all this, not only did we illustrate how to visualize any data source in Netdata using the StatsD protocol, but we have also created a new collector in the process.
```